yt_dlp/extractor/opencast.py

```diff
@@ -135,8 +138,10 @@ def _real_extract(self, url):
 
 class OpencastPlaylistIE(OpencastBaseIE):
     _VALID_URL = r'''(?x)
-                            https?://(?P<host>%s)/engage/ui/index.html\?.*?
-                            epFrom=(?P<id>%s)
+                            https?://(?P<host>%s)
+                            (/engage/ui/index.html\?.*?epFrom=|
+                            /ltitools/index.html\?.*?series=)
+                            (?P<id>%s)
                     ''' % (OpencastBaseIE._INSTANCES_RE, OpencastBaseIE._UUID_RE)
```

Comment:
```suggestion
    _VALID_URL = rf'''(?x)
        https?://(?P<host>{OpencastBaseIE._INSTANCES_RE})(
            /engage/ui/index.html\?(?:[^#]+&)?epFrom=|
            /lttools/index.html\?(?:[^#]+&)?series=
        )(?P<id>{OpencastBaseIE._UUID_RE})'''
```

Reply:
Thank you very much, I did not use rf strings before because vs-code does not highlight the regex colloring on such stings. Actually I even have opened an issue because of this https://github.com/microsoft/pylance-release/issues/2757 :D But, if you prefer these I'm fine with it.